translate/rus.json

```diff
@@ -329,7 +329,8 @@
 "SB":"КБ",
 "sched":"график",
 "schedule":"расписание",
-"score":"счёт личных встреч",
+"score":"счёт",
+"score~2":"оценка",  
```

Comment:
@octopoulos As I talked with you in Discord, ~2 meaning should be used on Photo 2. I will let you handle the code in that regard.